static/app/plugins/sessionstack/contexts/sessionstack.tsx

```diff
@@ -21,26 +22,24 @@ class SessionStackContextType extends Component<Props, State> {
   };
 
   componentDidMount() {
-    // eslint-disable-next-line react/no-find-dom-node
-    const domNode = findDOMNode(this) as HTMLElement;
-    this.parentNode = domNode.parentNode as HTMLElement;
     window.addEventListener('resize', () => this.setIframeSize(), false);
     this.setIframeSize();
   }
 
   componentWillUnmount() {
     window.removeEventListener('resize', () => this.setIframeSize(), false);
   }
-  parentNode?: HTMLElement;
+  iframeContainerId = uniqueId();
 
   getTitle = () => 'SessionStack';
 
   setIframeSize() {
-    if (this.state.showIframe || !this.parentNode) {
```

Comment:
i think this was backwards before? If showIframe then do nothing? width/height is only used on the iframe